src/Tests/dotnet-workload-update.Tests/GivenDotnetWorkloadUpdate.cs

```diff
@@ -397,6 +398,34 @@ public void ApplyRollbackWithMultipleManifestsAcrossFeatureBand()
             packInstaller.InstalledManifests[0].offlineCache.Should().Be(null);
         }
 
+        [Fact]
+        public void GivenInvalidVersionInRollbackFileItErrors()
+        {
+            _reporter.Clear();
+            
+            var testDirectory = _testAssetsManager.CreateTestDirectory().Path;
+            var dotnetRoot = Path.Combine(testDirectory, "dotnet");
+            var userProfileDir = Path.Combine(testDirectory, "user-profile");
+            Directory.CreateDirectory(userProfileDir);
+
+            var sdkFeatureVersion = "6.0.200";
+            var workloadResolver = WorkloadResolver.CreateForTests(new MockManifestProvider(new[] { _manifestPath }), dotnetRoot, userLocal: false, userProfileDir);
+            var nugetDownloader = new MockNuGetPackageDownloader(dotnetRoot);
```

Comment:
I don't think you need these two lines anymore, it looks like those variables aren't being used.